centreon-broker/core/src/broker.proto

```diff
@@ -109,4 +120,6 @@ message BrokerStats {
   repeated ModuleStats modules = 7;
   repeated SqlConnectionStats connections = 8;
   ConflictManagerStats conflict_manager = 9;
+  //repeated FailoverStats failovers = 10;
+  repeated MuxerStats muxers = 10;
```

Comment:
Instead of repeated MuxerStats, it would be better to make a map of MuxerStats indexed by their name.
Otherwise, we don't have any information about which muxer it is about.